src/pages/FilmPage.js

```diff
@@ -80,10 +84,15 @@ class FilmPage extends Component {
       this.searchRelated()
   }
 
+  static fetchData(store, match) {
```

Comment:
It's better to avoid keeping two functions with the same name but the different "type"